amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -557,33 +623,33 @@ Deletes the messages in a queue specified by the queue URL.
 |Type |String
```

Comment:
Same applies here, String returned by this operation contains ID of the request, shouldn't we mention it here?